importer/MtDe.py

```diff
@@ -9,13 +9,21 @@ def set_adm_location(self):
         """
         Set the administrative location.
 
-        Use the 'gemeinde' field as
-        municipality.
+        Use the 'gemeinde' field to resolve
+        municipality via wp link first, then if unable,
+        region-iso code via mapping.
         """
+        adm_q = None
         municip_raw = self.gemeinde
-        municip_q = utils.q_from_wikipedia("de", municip_raw)
-        if municip_q:
-            self.add_statement("located_adm", municip_q)
+        adm_q = utils.q_from_wikipedia("de", municip_raw)
+        if adm_q is None:
+            adm_iso_match = [x for x in self.data_files[
+                "regions"] if x["iso"] == self.region_iso]
```

Comment:
you called the file "councils" not "regions"

Reply:
Done.